packages/test/end-to-end-tests/package.json

```diff
@@ -26,10 +26,11 @@
     "start:tinylicious:test": "tinylicious > tinylicious.log 2>&1",
     "test": "npm run test:realsvc",
     "test:realsvc": "npm run test:realsvc:local && npm run test:realsvc:tinylicious",
-    "test:realsvc:local": "mocha dist/test --config src/test/.mocharc.js",
+    "test:realsvc:local": "cross-env FLUID_TEST_DRIVER= npm run test:realsvc:run -- --parallel",
```

Comment:
Missed setting the environment variable?

Reply:
It clears it on purpose.

Reply:
Although I think we accept "local" too.... so that might be less confusing.  I will change it.